dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityMessages.nlsprops

```diff
@@ -86,6 +86,16 @@ encode.option-desc.custom.description=\
 # encryption is enabled.
 no.custom.encyption=The custom password encryption is not detected.
 
+# The following messages are for retrieving an encryption key from a 
+# SAF keyring.
+saf.label.does.not.exist=The {0} label does not exist in the {1} keyring.  Unable to find a valid key for AES password encryption.
+saf.cert.is.not.key=The {0} certificate is not a key entry.  The certificate needs to be a key entry for use as an AES password encryption key.
```

Comment:
suggested edit

The certificate needs to be a key entry for use as an AES password encryption key.
--->
The certificate must be a key entry to use it as an AES password encryption key.